.devcontainer/.env.sample

```diff
@@ -12,32 +9,9 @@ AWS_BUCKET=bucket-name
 CONFIG_FILE=??_*.json
 
 # Django config
-DJANGO_ADMIN=false
-DJANGO_ALLOWED_HOSTS=localhost,127.0.0.1,[::1]
-DJANGO_TRUSTED_ORIGINS=http://localhost,http://127.0.0.1
-DJANGO_CSP_CONNECT_SRC=https://example.com
-DJANGO_CSP_FONT_SRC=https://california.azureedge.net/cdt/statetemplate/,https://fonts.gstatic.com/
-DJANGO_CSP_FRAME_SRC=https://example.com
-DJANGO_CSP_SCRIPT_SRC=https://code.jquery.com/,https://cdn.amplitude.com/,https://california.azureedge.net/
-DJANGO_CSP_STYLE_SRC=https://california.azureedge.net/cdt/statetemplate/,https://fonts.googleapis.com/css
-DJANGO_DB=django
-DJANGO_DEBUG=true
 DJANGO_INIT_PATH=fixtures/??_*.json
 DJANGO_LOCAL_PORT=8000
-DJANGO_LOG_LEVEL=DEBUG
-DJANGO_RECAPTCHA_API_URL=https://www.google.com/recaptcha/api.js
-DJANGO_RECAPTCHA_SITE_KEY=
-DJANGO_RECAPTCHA_SECRET_KEY=
-DJANGO_RECAPTCHA_VERIFY_URL=https://www.google.com/recaptcha/api/siteverify
 DJANGO_SECRET_KEY=secret
-DJANGO_RATE_LIMIT=0
-DJANGO_RATE_LIMIT_METHODS=GET,POST,PUT,DELETE
-DJANGO_RATE_LIMIT_PERIOD=0
-
-DJANGO_OAUTH_AUTHORITY=https://example.com
-DJANGO_OAUTH_CLIENT_ID=client-id-123
-DJANGO_OAUTH_CLIENT_NAME=client_name
-DJANGO_OAUTH_SCOPE=openid
 
```

Comment:
Should we keep these two in here?

```
# Django config
DJANGO_OAUTH_AUTHORITY=**** # CDT auth server root domain
DJANGO_OAUTH_CLIENT_ID=**** # client id for CDT auth server
```

Reply:
In my humble opinion, the ideal is for the app to Just Work™️ in development without any special configuration, so we shouldn't leave anything in the `.env.sample` file.

Reply:
@afeld Are you thinking that we shouldn't have `.env.sample` at all?

Reply:
That's the (eventual?) goal I'm suggesting, yep.

Reply:
I think we could actually get there if it weren't for the `DJANGO_LOCAL_PORT`. Thoughts on hardcoding to `11369` for the CDT auth server?

The others can be set in the `compose.yml` file directly.

Then maybe `.env.sample` goes back to being purely an example of all the possible environment variables, but we ditch the docs that instruct developers to first create an `.env` from it?

Reply:
> Thoughts on hardcoding to 11369 for the CDT auth server?

I'd be ok with that as the default.

> Then maybe .env.sample goes back to being purely an example of all the possible environment variables, but we ditch the docs that instruct developers to first create an .env from it?

Sounds good. I'd also be ok with `.env.sample` not having _all_ the possible environment variables, but rather ones that could be helpful to override maybe? I don't feel strongly either way, just an idea.